setup.py

```diff
@@ -73,7 +73,9 @@
     },
 
     entry_points = {
-        # Command-line programs we want setuptools to install:
+        # These are the command-line programs we want setuptools to install.
+        # Don't forget to modify the omnibus packaging tool
+        # (admin/packaging.py) if you make changes here.
```

Comment:
I guess we could have the list of entrypoints centrally, maybe in `admin/somewhere` which can be imported in `setup.py` and in the `packaging` module and its tests....as a followup issue though.